argocd/iac/pulumi/typescript/README.md

```diff
@@ -0,0 +1,11 @@
+# Pulumi Typescript GitOps Bridge
```

Comment:
Can you put the example in an `eks` directory?
`argocd/iac/pulumi/eks/typescript/README.md`